webapp/src/pages/Map.tsx

```diff
@@ -654,108 +666,152 @@ function MapPage(props: MapPageProps) {
+          break
+        case 'Display RSU Status':
+          dispatch(setDisplay({ view: 'tab', display: 'displayRsuErrors' }))
+          break
+      }
+    }
+  }
+
   return (
     <div className="container">
-      <Grid container className="legend-grid" direction="row">
-        <Legend />
-        {activeLayers.includes('rsu-layer') && (
-          <div className="rsu-status-div">
-            <h1 className="legend-header">RSU Status</h1>
-            <label className="rsu-status-label">
-              <input
-                className="rsu-status-input"
-                type="radio"
-                name="none-status-radio"
-                value="none"
-                checked={displayType === ''}
-                onChange={handleRsuDisplayTypeChange}
-              />
-              None
-            </label>
-
-            <label className="rsu-status-label">
-              <input
-                className="rsu-status-input"
-                type="radio"
-                name="online-status-radio"
-                value="online"
-                checked={displayType === 'online'}
-                onChange={handleRsuDisplayTypeChange}
-              />
-              Online Status
-            </label>
-
-            <label className="rsu-status-label">
-              <input
-                className="rsu-status-input"
-                type="radio"
-                name="scms-status-radio"
-                value="scms"
-                checked={displayType === 'scms'}
-                onChange={handleRsuDisplayTypeChange}
+      <div className="menu-container">
+        <Accordion disableGutters={true} className="menuAccordion" sx={{ '&.accordion': { marginBottom: 0 } }}>
+          <AccordionSummary expandIcon={<ExpandMoreIcon />} aria-controls="panel1-content" id="panel1-header">
+            <Typography fontFamily="Arial, Helvetica, sans-serif" fontSize="medium">
+              Layers
+            </Typography>
+          </AccordionSummary>
+          <AccordionDetails>
+            <Legend />
+          </AccordionDetails>
+        </Accordion>
+        <Accordion disableGutters={true} className="menuAccordion" sx={{ '&.accordion': { marginBottom: 0 } }}>
+          <AccordionSummary expandIcon={<ExpandMoreIcon />} aria-controls="panel1-content" id="panel1-header">
+            <Typography fontFamily="Arial, Helvetica, sans-serif" fontSize="medium">
+              Map Controls
+            </Typography>
+          </AccordionSummary>
+          <AccordionDetails>
+            <List>
+              <ListItem disablePadding>
+                <ListItemButton
+                  onClick={() => handleMenuSelection('Display Message Counts')}
+                  sx={{
+                    backgroundColor: menuSelection.includes('Display Message Counts') ? '#1c1c1c' : '#3d3d3d',
+                    borderBottom: menuSelection.includes('Display Message Counts') ? '1px solid black' : 'none',
```

Comment:
When this gets merged with the MUI theming PR, we will want to pull these colors from the theme